tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/IgnoreAllAction.java

```diff
@@ -34,7 +36,7 @@ public IgnoreAllAction( ISelectionProvider provider )
     }
 
     @Override
-    protected IStatus runWithMarker( TaskProblem taskProblem, IMarker marker )
+    protected IStatus runWithMarker( Problem problem, IMarker marker )
```

Comment:
It seems that we aren't using TaskProblem anymore, so we need to delete it completely.